server/athenian/api/models/web/filtered_code_check_run.py

```diff
@@ -0,0 +1,188 @@
+
+        :param title: The title of this FilteredCodeCheckRun.
+        """
+        if title is None:
+            raise ValueError("Invalid value for `title`, must not be `None`")
+
+        self._title = title
+
+    @property
+    def repository(self) -> str:
+        """Gets the repository of this FilteredCodeCheckRun.
+
+        Repository name which uniquely identifies any repository in any service provider.
+        The format matches the repository URL without the protocol part. No \".git\" should be
+        appended. We support a special syntax for repository sets: \"{reposet id}\" adds all
+        the repositories from the given set.
+
+        :return: The repository of this FilteredCodeCheckRun.
+        """
+        return self._repository
+
+    @repository.setter
+    def repository(self, repository: str):
+        """Sets the repository of this FilteredCodeCheckRun.
+
+        Repository name which uniquely identifies any repository in any service provider.
+        The format matches the repository URL without the protocol part. No \".git\" should be
+        appended. We support a special syntax for repository sets: \"{reposet id}\" adds all
+        the repositories from the given set.
+
+        :param repository: The repository of this FilteredCodeCheckRun.
+        """
+        if repository is None:
+            raise ValueError("Invalid value for `repository`, must not be `None`")
+
+        self._repository = repository
+
+    @property
+    def last_execution(self) -> datetime:
+        """Gets the last_execution of this FilteredCodeCheckRun.
+
+        Timestamp of when the check run launched last time.
+
+        :return: The last_execution of this FilteredCodeCheckRun.
+        """
+        return self._last_execution
+
+    @last_execution.setter
+    def last_execution(self, last_execution: datetime):
+        """Sets the last_execution of this FilteredCodeCheckRun.
+
+        Timestamp of when the check run launched last time.
+
+        :param last_execution: The last_execution of this FilteredCodeCheckRun.
+        """
+        if last_execution is None:
+            raise ValueError("Invalid value for `last_execution`, must not be `None`")
+
+        self._last_execution = last_execution
+
+    @property
+    def total_stats(self) -> CodeCheckRunStatistics:
+        """Gets the total_stats of this FilteredCodeCheckRun.
+
+        :return: The total_stats of this FilteredCodeCheckRun.
+        """
+        return self._total_stats
+
+    @total_stats.setter
+    def total_stats(self, total_stats: CodeCheckRunStatistics):
+        """Sets the total_stats of this FilteredCodeCheckRun.
+
+        :param total_stats: The total_stats of this FilteredCodeCheckRun.
+        """
+        if total_stats is None:
+            raise ValueError("Invalid value for `total_stats`, must not be `None`")
+
+        self._total_stats = total_stats
+
+    @property
+    def prs_stats(self) -> CodeCheckRunStatistics:
```

Comment:
Ah ok so prs_stats is the stats for checks run on_pull_requests, and total_stats for checks run on_pull_requests and on_push, right?

Reply:
Correct.